packages/main/src/CalendarLegendStandardItem.ts

```diff
@@ -0,0 +1,100 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { ITabbable } from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import CalendarLegendStandardItemsType from "./types/StandardCalendarLegendItemsType.js";
+import {
+	CAL_LEGEND_TODAY_TEXT,
+	CAL_LEGEND_SELECTED_TEXT,
+	CAL_LEGEND_WORKING_DAY_TEXT,
+	CAL_LEGEND_NON_WORKING_DAY_TEXT,
+} from "./generated/i18n/i18n-defaults.js";
+
+import CalendarLegendStandardItemTemplate from "./generated/templates/CalendarLegendStandardItemTemplate.lit.js";
+
+// Styles
+import CalendarLegendStandardItemCss from "./generated/themes/CalendarLegendStandardItem.css.js";
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ * The <code>ui5-cal-legend-standard-item</code> represents a standard legend item used within the <code>ui5-calendar-legend</code> component.
+ *
+ * <h3>Custom Text Override</h3>
```

Comment:
This I would call the section "Custom Text " and add after the Usage segment. And, start with explaining that by default we display built-in texts, but you can always alter the text via..

Reply:
This JSDoc is a little bit outdated, and I most likely forgot to remove this statement, since we decided that the app developer would not be able to edit the text of the standard/default calendar legend items (**today**, **selected**, **working**, **non**-**working**) as they are provided by default.